apps/condo/pages/tls.tsx

```diff
@@ -0,0 +1,404 @@
+import enAndroid from '@condo/lang/en/pages/tls/android.md'
+import enIos from '@condo/lang/en/pages/tls/ios.md'
```

Comment:
Why this imports are still here? Aren't you migrated to `serverSideProps`?

Reply:
It's used in `getServerSideProps`

Reply:
Why? If I want add a new language, should I also add imports? Probably not...

Ideally it should work like a following:
1. Get a locale from request
2. Have a set of file paths "tsl/file1.md", "tls/file2.md" and a root to lang directory (like `@app/condo/lang`)
3. Try to get content of `${rootPath}/${locale$}/${path}` if not found, use default locale

So you can get rid of any imports, next module declarations, and new lang-link

Reply:
Changed